docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,153 @@
+.. _acsincome-data:
+Retiring Adult: ACSIncome
```

Comment:
I believe their paper title was meant to be a pun about "retiring" the UCI adult dataset, so I'm not sure that it makes sense to include that pun as the title here, since it may make readers think it's a dataset about retiring adults..

Reply:
Fair point. I'll just say ACSIncome here to improve clarity. Thanks!